tst/testinstall/hpc/tasks.tst

```diff
@@ -1,11 +1,11 @@
 #############################################################################
 ##
-#W  task.tst                 GAP tests                   Alexander Konovalov
+#W  tasks.tst                 GAP tests                   Alexander Konovalov
 ##
 ##
 #Y  Copyright (C)  2012
 ##
-gap> START_TEST("task.tst");
+gap> START_TEST("tasks.tst");
```

Comment:
Any particular reason for renaming the file (which increases the diffs in this PR but otherwise seems to serve no purpose)?

Reply:
It describes the content of the file better and is more consistent with the naming of the other file in the directory. But I don't insist on the changes...